server/src/concepts/user/useCases/createNewUser.js

```diff
@@ -3,7 +3,7 @@ import bcrypt from "bcrypt";
 import { getUserWithEmail } from "../repositories/queries.js";
 import sendEmail from "../../../services/mail/index.js";
 
-const createHashedPassword = async (password) => {
+export const createHashedPassword = async (password) => {
```

Comment:
we can move this method into some other directory in future. Maybe into concepts/auth or concepts/user/helpers. It would be better approach